bears/css/CSSCombBear.py

```diff
@@ -0,0 +1,28 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='csscomb',
+        output_format='corrected',
+        use_stdout=True,
+        use_stdin=False,
+        result_message='Change coding style based on CSS rules written in ' +
+                       '.csscomb.json',
+        )
+class CSSCombBear:
+    """
+    CSScomb is a coding style formatter for CSS.
+
+    For more information, consult <https://github.com/csscomb/csscomb.js>.
+    """
+
+    LANGUAGES = {"CSS"}
+    REQUIREMENTS = {NpmRequirement('csscomb')}
```

Comment:
heh did it feel good to write that adrian? :P

On Wed, 31 Aug 2016 at 17:41 Zatreanu Adrian-Gabriel <
notifications@github.com> wrote:

> In bears/css/CSSCombBear.py
> <https://github.com/coala-analyzer/coala-bears/pull/732#discussion_r77010862>
> :
>
> > +@linter(executable='csscomb',
> > +        output_format='corrected',
> > +        use_stdout=True,
> > +        use_stdin=False,
> > +        result_message='Change coding style based on CSS rules written in ' +
> > +                       '.csscomb.json',
> > +        )
> > +class CSSCombBear:
> > +    """
> > +    CSScomb is a coding style formatter for CSS.
> > +
> > +    For more information, consult <https://github.com/csscomb/csscomb.js>.
> > +    """
> > +
> > +    LANGUAGES = {"CSS"}
> > +    REQUIREMENTS = {NpmRequirement('csscomb')}
>
> no necessary version?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/732/files/0b6b54e3431846185eec57b26846dfc3e0f3c2ea#r77010862>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFc6KH31QWSaufZw4mDVancIMNyiLaQjks5qlZ7cgaJpZM4Jv2yy>
> .
>